packages/wrangler/src/__tests__/d1/export.test.ts

```diff
@@ -117,10 +118,16 @@ describe("execute", () => {
 			http.post(
 				"*/accounts/:accountId/d1/database/:databaseId/export",
 				async ({ request }) => {
+					// This endpoint is polled recursively. If we respond immediately,
+					// the callstack builds up quickly leading to a hard-to-debug OOM error.
+					// This timeout ensures that if the endpoint is accidently polled infinitely
+					// the test will timeout before breaching available memory
+					await setTimeout(10);
+
 					const body = (await request.json()) as Record<string, unknown>;
 
 					// First request, initiates a new task
-					if (!body.currentBookmark) {
+					if (!body.current_bookmark) {
```

Comment:
This missed renaming from `currentBookmark` to `current_bookmark` was the culprit of the OOM

I have also added a timeout above to avoid this happening in the future. The OOM error was very opaque and hard to debug with many red herrings.

Reply:
u da best Rahul <3!!!